### ISSUE TITLE
Question for PN2.py

### ISSUE THREAD

**Issue by QueYiXin** [NOrangeeroli/SecondPose#18]
I can't understand the code in [(https://github.com/NOrangeeroli/SecondPose/blob/main/model/PN2.py)](url) as following:
`39  x = torch.cat([pts, pts, rgb], dim=2)`
`40  x = self.pn2msg(x)`

Why the x is composed with two 'pts'.
